manifests/charts/gateways/istio-ingress/templates/poddisruptionbudget.yaml

```diff
@@ -16,7 +16,12 @@ metadata:
     install.operator.istio.io/owning-resource: {{ .Values.ownerName | default "unknown" }}
     operator.istio.io/component: "IngressGateways"
 spec:
-  minAvailable: 1
+  {{- if .Values.minAvailable }}
```

Comment:
The `gateways` chart is considered deprecated, and we will not be including new features in this chart. Could you please try using the `gateway` chart to see if it resolves your issue?